### PR TITLE
[docs-only] Update Gateway Readme

### PR DESCRIPTION
**Pull request by kobergj**
Improves the gateway readme

### CODE REVIEW

File: services/gateway/README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # Gateway
 
-The gateway service is an ...
+The gateway service is responsible for passing requests to the storage providers. Other services never talk to the storage providers directly but will always send their requests to the `gateway` service.
```
Comment (mmattel) [inline services/gateway/README.md:3]: 🧡

File: services/gateway/README.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 
 ## Caching
 
-The `gateway` service can use a configured store via `GATEWAY_STAT_CACHE_STORE`. Possible stores are:
+The gateway services is using caching as it is highly frequented with the same requests. As of now it uses two different caches:
+  -   the `provider cache` is caching requests to list or get storage providers.
+  -   the `create home cache` is caching requests to create personal spaces (as they only need to be executed once).
+
+Both caches can be configured via the `OCIS_CACHE_*` envvars (or `GATEWAY_PROVIDER_CACHE_*` and `GATEWAY_CREATE_HOME_CACHE_*` respectively). See envvar section for details. @mmattel how to link?
```
Comment (mmattel) [inline services/gateway/README.md:11]: Will check

File: services/gateway/README.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
+The gateway service is responsible for passing requests to the storage providers. Other services never talk to the storage providers directly but will always send their requests to the `gateway` service.
 
 ## Caching
 
-The `gateway` service can use a configured store via `GATEWAY_STAT_CACHE_STORE`. Possible stores are:
+The gateway services is using caching as it is highly frequented with the same requests. As of now it uses two different caches:
+  -   the `provider cache` is caching requests to list or get storage providers.
+  -   the `create home cache` is caching requests to create personal spaces (as they only need to be executed once).
+
+Both caches can be configured via the `OCIS_CACHE_*` envvars (or `GATEWAY_PROVIDER_CACHE_*` and `GATEWAY_CREATE_HOME_CACHE_*` respectively). See the [envvar section](/services/gateway/configuration/#environment-variables) for details.
```
Comment (mmattel) [inline services/gateway/README.md:11]: Link fixed locally, checked and pushed.

File: services/gateway/README.md (hunk 1 above)
Comment (EParzefall) [reply services/gateway/README.md:3]: I'd say "via" is more fitting, although it's implied that the requests will reach the storage providers in the end.

File: services/gateway/README.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
 # Gateway
 
-The gateway service is an ...
+The gateway service is responsible for passing requests to the storage providers. Other services never talk to the storage providers directly but will always send their requests to the `gateway` service.
 
 ## Caching
 
-The `gateway` service can use a configured store via `GATEWAY_STAT_CACHE_STORE`. Possible stores are:
+The gateway services is using caching as it is highly frequented with the same requests. As of now it uses two different caches:
```
Comment (EParzefall) [inline services/gateway/README.md:7]:
```suggestion
The gateway service is using caching as it is highly frequented with the same requests. As of now it uses two different caches:
```

File: services/gateway/README.md (hunk 1 above)
Comment (mmattel) [inline services/gateway/README.md:3]:
```suggestion
The gateway service is responsible for passing requests to the storage providers. Other services never talk to the storage providers directly but will always send their requests via the `gateway` service.
```

**Review by mmattel — APPROVED** [review]

**Pull request merged**